Middlewares/Third_Party/LwIP/system/OS/sys_arch.c

```diff
@@ -41,9 +41,6 @@
 
 #include "cmsis_os.h"
 
-#if defined(LWIP_PROVIDE_ERRNO)
```

Comment:
These lines should be updated as show below rather than removed.

```c
#if (defined (__CC_ARM) || defined (__ARMCC_VERSION) || defined (__ICCARM__))
int errno;
#endif
```